identifiers/country-au/councils.csv

```diff
@@ -1,30 +1,30 @@
-id,name
-ocd-division/country:au/state:wa/sldu:agricultural,Agricultural Region
-ocd-division/country:au/state:tas/sldu:derwent,Derwent
-ocd-division/country:au/state:wa/sldu:east_metropolitan,East Metropolitan Region
-ocd-division/country:au/state:vic/sldu:eastern_victoria,Eastern Victoria
-ocd-division/country:au/state:tas/sldu:elwick,Elwick
-ocd-division/country:au/state:tas/sldu:hobart,Hobart
-ocd-division/country:au/state:tas/sldu:huon,Huon
-ocd-division/country:au/state:tas/sldu:launceston,Launceston
-ocd-division/country:au/state:tas/sldu:mcintyre,McIntyre
-ocd-division/country:au/state:tas/sldu:mersey,Mersey
-ocd-division/country:au/state:wa/sldu:mining_pastoral,Mining and Pastoral Region
-ocd-division/country:au/state:tas/sldu:montgomery,Montgomery
-ocd-division/country:au/state:tas/sldu:murchison,Murchison
-ocd-division/country:au/state:tas/sldu:nelson,Nelson
-ocd-division/country:au/state:vic/sldu:north-eastern_metropolitan,North - Eastern Metropolitan Region
-ocd-division/country:au/state:wa/sldu:north_metropolitan,North Metropolitan Region
-ocd-division/country:au/state:vic/sldu:northern_metropolitan,Northern Metropolitan
-ocd-division/country:au/state:vic/sldu:northern_victoria,Northern Victoria
-ocd-division/country:au/state:tas/sldu:pembroke,Pembroke
-ocd-division/country:au/state:tas/sldu:prosser,Prosser
-ocd-division/country:au/state:tas/sldu:rosevears,Rosevears
-ocd-division/country:au/state:tas/sldu:rumney,Rumney
-ocd-division/country:au/state:vic/sldu:south-eastern_metropolitan,South Eastern Metropolitan
-ocd-division/country:au/state:wa/sldu:south_metropolitan,South Metropolitan Region
-ocd-division/country:au/state:wa/sldu:south_west,South West Region
-ocd-division/country:au/state:vic/sldu:southern_metropolitan,Southern Metropolitan
-ocd-division/country:au/state:vic/sldu:western_metropolitan,Western Metropolitan
-ocd-division/country:au/state:vic/sldu:western_victoria,Western Victoria
-ocd-division/country:au/state:tas/sldu:windermere,Windermere
+id,name,validFrom,validThrough
```

Comment:
Can you provide documentation to confirm these changes?

Reply:
@HKSenior These changes in Victoria and Tasmania are not made by us, wrong ocd ids are being referred here. Kindly guide us which changes you are referring to.

Reply:
I see these changes are solely reorganizing into alphabetical order.

In the future it would be best to submit 2 PRs one with the updates due to the upcoming redistricting and another as cleanup. The sorting is nice to have but can bloat diffs and make discerning between cleanup and actual changes a bit difficult to spot.